docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -690,7 +690,7 @@ Status Codes:
 
 ### Resize a container TTY
 
-`POST /containers/(id)/resize?h=<height>&w=<width>`
+`POST /containers/(id)/resize`
```

Comment:
Oh, sorry, this slipped my attention. I think it's good like this; the example below shows how to use it, which should explain its use.